requirements.txt

```diff
@@ -4,3 +4,4 @@ numpy>=1.19.5
 Pillow>=8.2.0
 python-magic>=0.4.22
 regex>=2021.4.4
+cupy>=9.6.0
```

Comment:
From the test workflow failure it looks like you can't install `cupy` if you don't have CUDA support. We don't want to make it so people can't install `augly` if they don't have CUDA. Maybe one way to solve this is to:
- Make this requirement optional (you can add an `extra_requires` to the `setup.py` file, like we already have for `av` but called `gpu` which contains this cupy requirement)
- Wrap the `import cupy as cp` in a `try` statement and if it succeeds, then set a var like `cupy = True`. Then only convert the numpy arrays to cupy & manipulate them using cupy instead of numpy if `cupy == True`
- Also, let's not require the most recent version of `cupy` if we don't have to. You can just require `cupy`, or find a pretty recent version that has the functions that you use from `cupy`